app/Http/Controllers/Api/UsersController.php

```diff
@@ -74,6 +83,17 @@ public function store(UserRequest $request)
         }
 
         $contest->waitingRoom->users()->attach($user->id);
+        $this->manager->appendCampaign($contest);
+
+        // Fire off welcome Email
+        $message = Message::where(['contest_id' => $contest->id, 'type' => 'welcome'])->first();
```

Comment:
I mean it could be, I'm not sure if theres much benefit in it though.
- this is (AFAIK) the only place that needs the welcome email
- its 1 line of code, so its not breaking much up